src/issues/util.ts

```diff
@@ -113,13 +113,13 @@ export function userMarkdown(origin: PullRequestDefaults, user: User): vscode.Ma
 		markdown.appendMarkdown('  \r\n\r\n---');
 	}
 	if (user.location) {
-		markdown.appendMarkdown(`  \r\n$(location) ${user.location}`);
+		markdown.appendMarkdown(`  \r\n${vscode.l10n.t('{0} {1}', '$(location)', user.location)}`);
```

Comment:
doesn't really need localization does it

Reply:
Does this not need localization for RTL languages so that the icon will be on the correct side?

Reply:
We don't support [any RTL languages](https://github.com/microsoft/vscode/issues/11770) but that's a good point. Might as well leave it